downstream/modules/platform/ref-eda-system-requirements.adoc

```diff
@@ -2,7 +2,13 @@
 
 = {EDAcontroller} system requirements
 
-The {EDAcontroller} is a single-node system capable of handling a variable number of long-running processes (such as, Rulebook activations) on-demand, depending on the number of CPU cores. Use the following minimum requirements to execute a maximum of 9 simultaneous activations:
+The {EDAcontroller} is a single-node system capable of handling a variable number of long-running processes (such as, Rulebook activations) on-demand, depending on the number of CPU cores. Use the following minimum requirements to execute, by default, a maximum of 12 simultaneous activations:
```

Comment:
For better readability, I would move the two admonitions ("Important" and "Note") to after the minimum requirements table. You have a colon at the end of the sentence, which indicates that the minimum requirements you referenced should come next.

Reply:
I considered that, but thought notes primarily came before the relevant content. As for the arrangement of the admonitions themselves, I'll follow the example in the Deploying AAP operator on OCP Guide in Chapter 6 (bullet list under "IMPORTANT"). Will send back shortly.